libraries/tickv/src/crc32.rs

```diff
@@ -307,15 +309,11 @@ const fn crc32(poly: u32, mut byte: u8) -> u32 {
     [value, reflect_32(value)][0]
 }
 
-pub struct Crc {}
-
-pub struct Digest<'a> {
-    crc: &'a Crc,
-    value: u32,
-}
+struct Crc {}
 
 impl Crc {
```

Comment:
Can you document the distinction between Crc32 and Crc, and why they are separated?

Reply:
Done